--- candidate
+++ reference (mapped)
@@ -1,1 +1,1 @@
-visible 50535
+visible 50555
@@ -1545,1 +1545,1 @@
-            "path": "App\\Big",
+            "path": "src/m45/Big/C1.php",
@@ -1557,1 +1557,1 @@
-            "path": "App\\Small",
+            "path": "src/m45/Small/C1.php",
@@ -1576,1 +1576,1 @@
-invocation-format:gitlab 50535
+invocation-format:gitlab 50555
@@ -3120,1 +3120,1 @@
-            "path": "App\\Big",
+            "path": "src/m45/Big/C1.php",
@@ -3132,1 +3132,1 @@
-            "path": "App\\Small",
+            "path": "src/m45/Small/C1.php",
