--- candidate
+++ reference (mapped)
@@ -1,1 +1,1 @@
-visible 49770
+visible 50535
@@ -1540,0 +1540,24 @@
+        "description": "Class count is 30, exceeds threshold of 25. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "0a232c91a79487df0b2efdd94459aae7",
+        "severity": "critical",
+        "location": {
+            "path": "App\\Big",
+            "lines": {
+                "begin": 1
+            }
+        }
+    },
+    {
+        "description": "Class count is 16, exceeds threshold of 15. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "7a216416493f788032750da54ff9ec62",
+        "severity": "major",
+        "location": {
+            "path": "App\\Small",
+            "lines": {
+                "begin": 1
+            }
+        }
+    },
+    {
@@ -1552,1 +1576,1 @@
-invocation-format:gitlab 49770
+invocation-format:gitlab 50535
@@ -3087,0 +3111,24 @@
+            }
+        }
+    },
+    {
+        "description": "Class count is 30, exceeds threshold of 25. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "0a232c91a79487df0b2efdd94459aae7",
+        "severity": "critical",
+        "location": {
+            "path": "App\\Big",
+            "lines": {
+                "begin": 1
+            }
+        }
+    },
+    {
+        "description": "Class count is 16, exceeds threshold of 15. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "7a216416493f788032750da54ff9ec62",
+        "severity": "major",
+        "location": {
+            "path": "App\\Small",
+            "lines": {
+                "begin": 1
