--- candidate
+++ reference (mapped)
@@ -1,1 +1,1 @@
-visible 5954
+visible 7141
@@ -124,0 +124,36 @@
+        "description": "Class count is 7, exceeds threshold of 6. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "97d1969b76aa71a568be06a8cd004ab1",
+        "severity": "critical",
+        "location": {
+            "path": "Corpus\\Design\\Anonymous",
+            "lines": {
+                "begin": 1
+            }
+        }
+    },
+    {
+        "description": "Class count is 9, exceeds threshold of 6. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "18b9259b298b793411fe0a70a6f818b7",
+        "severity": "critical",
+        "location": {
+            "path": "Corpus\\Design\\Hierarchy",
+            "lines": {
+                "begin": 1
+            }
+        }
+    },
+    {
+        "description": "Class count is 3, exceeds threshold of 2. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "c68d829739ebdde1fc24bb935986428b",
+        "severity": "major",
+        "location": {
+            "path": "Corpus\\Design\\Model",
+            "lines": {
+                "begin": 1
+            }
+        }
+    },
+    {
@@ -172,1 +208,1 @@
-invocation-format:gitlab 5954
+invocation-format:gitlab 7141
@@ -291,0 +327,36 @@
+            }
+        }
+    },
+    {
+        "description": "Class count is 7, exceeds threshold of 6. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "97d1969b76aa71a568be06a8cd004ab1",
+        "severity": "critical",
+        "location": {
+            "path": "Corpus\\Design\\Anonymous",
+            "lines": {
+                "begin": 1
+            }
+        }
+    },
+    {
+        "description": "Class count is 9, exceeds threshold of 6. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "18b9259b298b793411fe0a70a6f818b7",
+        "severity": "critical",
+        "location": {
+            "path": "Corpus\\Design\\Hierarchy",
+            "lines": {
+                "begin": 1
+            }
+        }
+    },
+    {
+        "description": "Class count is 3, exceeds threshold of 2. Consider splitting into sub-namespaces",
+        "check_name": "size.class-count",
+        "fingerprint": "c68d829739ebdde1fc24bb935986428b",
+        "severity": "major",
+        "location": {
+            "path": "Corpus\\Design\\Model",
+            "lines": {
+                "begin": 1
