--- candidate
+++ reference (mapped)
@@ -500,1 +500,1 @@
-                "design.dit.avg": 0.7647058823529411,
+                "design.dit.avg": 1,
@@ -529,2 +529,2 @@
-                "design.noc.sum": 7,
-                "design.noc.avg": 0.3888888888888889,
+                "design.noc.sum": 9,
+                "design.noc.avg": 0.5,
@@ -723,3 +723,3 @@
-                "design.dit.avg": 0.14285714285714285,
-                "design.dit.max": 1,
-                "design.dit.p95": 0.6999999999999993,
+                "design.dit.avg": 0.7142857142857143,
+                "design.dit.max": 2,
+                "design.dit.p95": 1.6999999999999993,
@@ -761,2 +761,2 @@
-                "design.noc.sum": 1,
-                "design.noc.avg": 0.125,
+                "design.noc.sum": 3,
+                "design.noc.avg": 0.375,
@@ -1408,1 +1408,1 @@
-                "design.dit.avg": 0.7647058823529411,
+                "design.dit.avg": 1,
@@ -1446,2 +1446,2 @@
-                "design.noc.sum": 7,
-                "design.noc.avg": 0.3888888888888889,
+                "design.noc.sum": 9,
+                "design.noc.avg": 0.5,
@@ -1627,1 +1627,1 @@
-                "design.dit.avg": 0.7647058823529411,
+                "design.dit.avg": 1,
@@ -1665,2 +1665,2 @@
-                "design.noc.sum": 7,
-                "design.noc.avg": 0.3888888888888889,
+                "design.noc.sum": 9,
+                "design.noc.avg": 0.5,
@@ -1770,1 +1770,1 @@
-                "design.dit": 0,
+                "design.dit": 1,
@@ -1908,1 +1908,1 @@
-                "design.dit": 0,
+                "design.dit": 2,
@@ -2031,1 +2031,1 @@
-                "design.noc": 0,
+                "design.noc": 1,
@@ -2046,1 +2046,1 @@
-                "design.dit": 0,
+                "design.dit": 1,
@@ -2169,1 +2169,1 @@
-                "design.noc": 0,
+                "design.noc": 1,
