--- candidate
+++ reference (mapped)
@@ -1140,1 +1140,1 @@
-                        "ruleName": "health.complexity",
+                        "ruleName": "computed.health",
@@ -1153,1 +1153,1 @@
-                        "ruleName": "health.cohesion",
+                        "ruleName": "computed.health",
@@ -1166,1 +1166,1 @@
-                        "ruleName": "health.typing",
+                        "ruleName": "computed.health",
@@ -1179,1 +1179,1 @@
-                        "ruleName": "health.maintainability",
+                        "ruleName": "computed.health",
@@ -1192,1 +1192,1 @@
-                        "ruleName": "health.overall",
+                        "ruleName": "computed.health",
@@ -1205,1 +1205,1 @@
-                        "ruleName": "computed",
+                        "ruleName": "computed.health",
@@ -1447,1 +1447,1 @@
-                                "ruleName": "health.complexity",
+                                "ruleName": "computed.health",
@@ -1460,1 +1460,1 @@
-                                "ruleName": "health.cohesion",
+                                "ruleName": "computed.health",
@@ -1473,1 +1473,1 @@
-                                "ruleName": "health.typing",
+                                "ruleName": "computed.health",
@@ -1486,1 +1486,1 @@
-                                "ruleName": "health.maintainability",
+                                "ruleName": "computed.health",
@@ -1499,1 +1499,1 @@
-                                "ruleName": "health.overall",
+                                "ruleName": "computed.health",
@@ -1512,1 +1512,1 @@
-                                "ruleName": "computed",
+                                "ruleName": "computed.health",
@@ -1771,1 +1771,1 @@
-                                        "ruleName": "health.complexity",
+                                        "ruleName": "computed.health",
@@ -1784,1 +1784,1 @@
-                                        "ruleName": "health.cohesion",
+                                        "ruleName": "computed.health",
@@ -1797,1 +1797,1 @@
-                                        "ruleName": "health.coupling",
+                                        "ruleName": "computed.health",
@@ -1810,1 +1810,1 @@
-                                        "ruleName": "health.typing",
+                                        "ruleName": "computed.health",
@@ -1823,1 +1823,1 @@
-                                        "ruleName": "health.maintainability",
+                                        "ruleName": "computed.health",
@@ -1836,1 +1836,1 @@
-                                        "ruleName": "health.overall",
+                                        "ruleName": "computed.health",
@@ -1849,1 +1849,1 @@
-                                        "ruleName": "computed",
+                                        "ruleName": "computed.health",
@@ -1966,1 +1966,1 @@
-                                                "ruleName": "health.complexity",
+                                                "ruleName": "computed.health",
@@ -1979,1 +1979,1 @@
-                                                "ruleName": "health.coupling",
+                                                "ruleName": "computed.health",
@@ -1992,1 +1992,1 @@
-                                                "ruleName": "health.typing",
+                                                "ruleName": "computed.health",
@@ -2005,1 +2005,1 @@
-                                                "ruleName": "health.maintainability",
+                                                "ruleName": "computed.health",
@@ -2018,1 +2018,1 @@
-                                                "ruleName": "health.overall",
+                                                "ruleName": "computed.health",
@@ -2031,1 +2031,1 @@
-                                                "ruleName": "computed",
+                                                "ruleName": "computed.health",
@@ -2148,1 +2148,1 @@
-                                                "ruleName": "health.complexity",
+                                                "ruleName": "computed.health",
@@ -2161,1 +2161,1 @@
-                                                "ruleName": "health.cohesion",
+                                                "ruleName": "computed.health",
@@ -2174,1 +2174,1 @@
-                                                "ruleName": "health.coupling",
+                                                "ruleName": "computed.health",
@@ -2187,1 +2187,1 @@
-                                                "ruleName": "health.typing",
+                                                "ruleName": "computed.health",
@@ -2200,1 +2200,1 @@
-                                                "ruleName": "health.maintainability",
+                                                "ruleName": "computed.health",
@@ -2213,1 +2213,1 @@
-                                                "ruleName": "health.overall",
+                                                "ruleName": "computed.health",
@@ -2226,1 +2226,1 @@
-                                                "ruleName": "computed",
+                                                "ruleName": "computed.health",
@@ -2487,1 +2487,1 @@
-                                        "ruleName": "health.complexity",
+                                        "ruleName": "computed.health",
@@ -2500,1 +2500,1 @@
-                                        "ruleName": "health.cohesion",
+                                        "ruleName": "computed.health",
@@ -2513,1 +2513,1 @@
-                                        "ruleName": "health.coupling",
+                                        "ruleName": "computed.health",
@@ -2526,1 +2526,1 @@
-                                        "ruleName": "health.typing",
+                                        "ruleName": "computed.health",
@@ -2539,1 +2539,1 @@
-                                        "ruleName": "health.maintainability",
+                                        "ruleName": "computed.health",
@@ -2552,1 +2552,1 @@
-                                        "ruleName": "health.overall",
+                                        "ruleName": "computed.health",
@@ -2565,1 +2565,1 @@
-                                        "ruleName": "computed",
+                                        "ruleName": "computed.health",
@@ -2682,1 +2682,1 @@
-                                                "ruleName": "health.typing",
+                                                "ruleName": "computed.health",
@@ -2695,1 +2695,1 @@
-                                                "ruleName": "health.maintainability",
+                                                "ruleName": "computed.health",
@@ -2708,1 +2708,1 @@
-                                                "ruleName": "health.overall",
+                                                "ruleName": "computed.health",
@@ -2721,1 +2721,1 @@
-                                                "ruleName": "computed",
+                                                "ruleName": "computed.health",
@@ -2838,1 +2838,1 @@
-                                                "ruleName": "health.cohesion",
+                                                "ruleName": "computed.health",
@@ -2851,1 +2851,1 @@
-                                                "ruleName": "health.typing",
+                                                "ruleName": "computed.health",
@@ -2864,1 +2864,1 @@
-                                                "ruleName": "health.maintainability",
+                                                "ruleName": "computed.health",
@@ -2877,1 +2877,1 @@
-                                                "ruleName": "health.overall",
+                                                "ruleName": "computed.health",
@@ -2890,1 +2890,1 @@
-                                                "ruleName": "computed",
+                                                "ruleName": "computed.health",
@@ -3007,1 +3007,1 @@
-                                                "ruleName": "health.typing",
+                                                "ruleName": "computed.health",
@@ -3020,1 +3020,1 @@
-                                                "ruleName": "health.maintainability",
+                                                "ruleName": "computed.health",
@@ -3033,1 +3033,1 @@
-                                                "ruleName": "health.overall",
+                                                "ruleName": "computed.health",
@@ -3046,1 +3046,1 @@
-                                                "ruleName": "computed",
+                                                "ruleName": "computed.health",
@@ -3311,1 +3311,1 @@
-                        "ruleName": "health.complexity",
+                        "ruleName": "computed.health",
@@ -3324,1 +3324,1 @@
-                        "ruleName": "health.coupling",
+                        "ruleName": "computed.health",
@@ -3337,1 +3337,1 @@
-                        "ruleName": "health.typing",
+                        "ruleName": "computed.health",
@@ -3350,1 +3350,1 @@
-                        "ruleName": "health.maintainability",
+                        "ruleName": "computed.health",
@@ -3363,1 +3363,1 @@
-                        "ruleName": "health.overall",
+                        "ruleName": "computed.health",
@@ -3376,1 +3376,1 @@
-                        "ruleName": "computed",
+                        "ruleName": "computed.health",
@@ -3493,1 +3493,1 @@
-                                "ruleName": "health.complexity",
+                                "ruleName": "computed.health",
@@ -3506,1 +3506,1 @@
-                                "ruleName": "health.typing",
+                                "ruleName": "computed.health",
@@ -3519,1 +3519,1 @@
-                                "ruleName": "health.maintainability",
+                                "ruleName": "computed.health",
@@ -3532,1 +3532,1 @@
-                                "ruleName": "health.overall",
+                                "ruleName": "computed.health",
@@ -3545,1 +3545,1 @@
-                                "ruleName": "computed",
+                                "ruleName": "computed.health",
@@ -3806,1 +3806,1 @@
-                        "ruleName": "health.coupling",
+                        "ruleName": "computed.health",
@@ -3819,1 +3819,1 @@
-                        "ruleName": "health.typing",
+                        "ruleName": "computed.health",
@@ -3832,1 +3832,1 @@
-                        "ruleName": "health.overall",
+                        "ruleName": "computed.health",
@@ -3845,1 +3845,1 @@
-                        "ruleName": "computed",
+                        "ruleName": "computed.health",
@@ -3962,1 +3962,1 @@
-                                "ruleName": "health.typing",
+                                "ruleName": "computed.health",
@@ -3975,1 +3975,1 @@
-                                "ruleName": "health.maintainability",
+                                "ruleName": "computed.health",
@@ -3988,1 +3988,1 @@
-                                "ruleName": "health.overall",
+                                "ruleName": "computed.health",
@@ -4001,1 +4001,1 @@
-                                "ruleName": "computed",
+                                "ruleName": "computed.health",
