--- candidate
+++ reference (mapped)
@@ -398,1 +398,0 @@
-                "size.symbol-declaring-namespace-count": 7,
@@ -430,2 +429,2 @@
-                "coupling.distance-own.avg": 0.380952380952381,
-                "coupling.distance-own.count": 7
+                "coupling.distance.avg": 0.380952380952381,
+                "coupling.distance.count": 7
@@ -629,1 +628,0 @@
-                "coupling.abstractness-own": 0,
@@ -634,5 +632,1 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 4,
-                "coupling.instability-own": 1,
-                "coupling.distance": 0,
-                "coupling.distance-own": 0,
+                "coupling.distance": 0,
@@ -864,1 +858,0 @@
-                "coupling.abstractness-own": 0,
@@ -869,5 +862,1 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 1,
-                "coupling.instability-own": 1,
-                "coupling.distance": 0,
-                "coupling.distance-own": 0,
+                "coupling.distance": 0,
@@ -1099,1 +1088,0 @@
-                "coupling.abstractness-own": 0,
@@ -1104,5 +1092,1 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 2,
-                "coupling.instability-own": 1,
-                "coupling.distance": 0,
-                "coupling.distance-own": 0,
+                "coupling.distance": 0,
@@ -1334,1 +1318,0 @@
-                "coupling.abstractness-own": 0,
@@ -1339,5 +1322,1 @@
-                "coupling.ca-own": 1,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
-                "coupling.distance": 1,
-                "coupling.distance-own": 1,
+                "coupling.distance": 1,
@@ -1526,1 +1505,0 @@
-                "coupling.abstractness-own": 0.3333333333333333,
@@ -1531,5 +1509,1 @@
-                "coupling.ca-own": 4,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
-                "coupling.distance": 0.6666666666666667,
-                "coupling.distance-own": 0.6666666666666667,
+                "coupling.distance": 0.6666666666666667,
@@ -1761,1 +1735,0 @@
-                "coupling.abstractness-own": 0,
@@ -1766,5 +1739,1 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 1,
-                "coupling.instability-own": 1,
-                "coupling.distance": 0,
-                "coupling.distance-own": 0,
+                "coupling.distance": 0,
@@ -1996,1 +1965,0 @@
-                "coupling.abstractness-own": 0,
@@ -2001,5 +1969,1 @@
-                "coupling.ca-own": 2,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
-                "coupling.distance": 1,
-                "coupling.distance-own": 1,
+                "coupling.distance": 1,
@@ -2224,3 +2188,0 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
@@ -2446,3 +2407,0 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
@@ -3652,2 +3610,2 @@
-        "warnings": 3,
-        "info": 1
+        "warnings": 4,
+        "info": 0
