--- candidate
+++ reference (mapped)
@@ -248,1 +248,0 @@
-                "size.symbol-declaring-namespace-count": 1,
@@ -280,2 +279,2 @@
-                "coupling.distance-own.avg": 1,
-                "coupling.distance-own.count": 1
+                "coupling.distance.avg": 1,
+                "coupling.distance.count": 1
@@ -473,1 +472,0 @@
-                "coupling.abstractness-own": 0,
@@ -478,5 +476,1 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
-                "coupling.distance": 1,
-                "coupling.distance-own": 1,
+                "coupling.distance": 1,
@@ -695,3 +689,0 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
