--- candidate
+++ reference (mapped)
@@ -308,1 +308,0 @@
-                "size.symbol-declaring-namespace-count": 4,
@@ -340,2 +339,2 @@
-                "coupling.distance-own.avg": 0.75,
-                "coupling.distance-own.count": 4,
+                "coupling.distance.avg": 0.75,
+                "coupling.distance.count": 4,
@@ -546,1 +545,0 @@
-                "coupling.abstractness-own": 0,
@@ -551,5 +549,1 @@
-                "coupling.ca-own": 2,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
-                "coupling.distance": 1,
-                "coupling.distance-own": 1,
+                "coupling.distance": 1,
@@ -794,1 +788,0 @@
-                "coupling.abstractness-own": 0,
@@ -799,5 +792,1 @@
-                "coupling.ca-own": 2,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
-                "coupling.distance": 1,
-                "coupling.distance-own": 1,
+                "coupling.distance": 1,
@@ -1042,1 +1031,0 @@
-                "coupling.abstractness-own": 0,
@@ -1047,5 +1035,1 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 5,
-                "coupling.instability-own": 1,
-                "coupling.distance": 0,
-                "coupling.distance-own": 0,
+                "coupling.distance": 0,
@@ -1290,1 +1274,0 @@
-                "coupling.abstractness-own": 0,
@@ -1295,5 +1278,1 @@
-                "coupling.ca-own": 2,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
-                "coupling.distance": 1,
-                "coupling.distance-own": 1,
+                "coupling.distance": 1,
@@ -1525,3 +1504,0 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
@@ -1754,3 +1730,0 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
