--- candidate
+++ reference (mapped)
@@ -266,1 +266,0 @@
-                "size.symbol-declaring-namespace-count": 1,
@@ -298,2 +297,2 @@
-                "coupling.distance-own.avg": 1,
-                "coupling.distance-own.count": 1
+                "coupling.distance.avg": 1,
+                "coupling.distance.count": 1
@@ -491,1 +490,0 @@
-                "coupling.abstractness-own": 0,
@@ -496,5 +494,1 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
-                "coupling.distance": 1,
-                "coupling.distance-own": 1,
+                "coupling.distance": 1,
@@ -713,3 +707,0 @@
-                "coupling.ca-own": 0,
-                "coupling.ce-own": 0,
-                "coupling.instability-own": 0,
