--- candidate
+++ reference (mapped)
@@ -77,1 +77,1 @@
-        "fingerprint": "2269160dc9811e7b0b445c701e153ff0",
+        "fingerprint": "42ff3f8722694f5751f4be35814f279d",
@@ -89,1 +89,1 @@
-        "fingerprint": "6821650aa39d3c3d601ee9219b65a567",
+        "fingerprint": "3d7fc4178a5bb9e0f5d3807f77e7b518",
@@ -101,1 +101,1 @@
-        "fingerprint": "b79a4ccaea1870418a581eabccfa5ed1",
+        "fingerprint": "f483f07edfe3dfa2e359a37ff7b782ac",
